components/resident/HomeScreen.js

```diff
@@ -17,37 +17,23 @@ class HomeScreen extends React.Component {
     }
   }
 
-  hideSplashScreen = () => {
-    this.setState({
-      isVisible: false
-    })
-  }
   static navigationOptions = { title: 'HOME' }
 
   componentDidMount() {
-    var that = this
     this.props.getReports(this.props.user.id)
-    setTimeout(function () {
-      that.hideSplashScreen()
-    }, 3000)
+    this.props.getUpvotes(this.props.user.id)
```

Comment:
Should we move the getReports and getUpvotes to the MyPotholes component? I don't believe we're using them anymore for the home screen.